tests/utils_for_tests.py

```diff
@@ -0,0 +1,12 @@
+import torch
```

Comment:
Can you put this check into `tests/ignite/__init__.py` and import it as `from tests.ignite import is_mps_available_and_functional`